ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -164,6 +165,32 @@ objects:
           export TEST_PROVIDER='azure'
         fi
 
+        # create fips enable helper
+        function enable_fips() {
+            for name in $(oc get machineconfigpool --template='{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}')
+            do
+              cat > /tmp/fips-mc.yaml <<'EOF'
+        apiVersion: machineconfiguration.openshift.io/v1
+        kind: MachineConfig
+        metadata:
+          labels:
+            machineconfiguration.openshift.io/role: {{name}}
```

Comment:
You need to put quotes around `"{{name}}"` otherwise YAML will interpret this differently.